docs/USER-DOCUMENTATION.md

```diff
@@ -15,7 +15,7 @@ images, usually available from the image publishers themselves.
 Images known to require special treatment:
 
 - Microsoft Windows (use [Windows USB/DVD Download Tool][windows-usb-tool], or
-  [Rufus][rufus]).
+  [Rufus][rufus], or [WoeUSB][woeusb]).
```

Comment:
Can you remove the ", or" before the Rufus link? It reads a bit odd otherwise

Reply:
I suppose the comma-space shouldn't be removed?

Reply:
I've applied the request change (with comma-space keep intact), please review.